src/vmware/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Other changes:
 - Add `az vmware addon hcx` command group to manage a HCX addon
 - Add `az vmware addon srm` command group to manage a Site Recovery Manager (SRM) addon
 - Add `az vmware addon vr` command group to manage a vSphere Replication (VR) addon
+- Add `az vmware global-reach-connection` command group to manage Global Reach Connections
```

Comment:
I see there are many features in 3.0, but not released. Do you still have new features to add? If not, recommend to upgrade version in setup.py here. Or will you raise a new PR to upgrade the version in setup.py tp trigger our extension release pipeline?

Reply:
The new API is still behind a feature flag in production.  After the feature flag is removed, I'll create a new PR to upgrade the version in setup.py to trigger the release.